Library/Homebrew/rubocops/mixins/on_formula_class.rb

```diff
@@ -0,0 +1,18 @@
+module RuboCop
+  module Cop
+    # Helper module for checking formula class
+    module FormulaDef
+      def file_inside_formula_root?
+        return false unless processed_source.path =~ %r{#{ENV["HOMEBREW_LIBRARY"]}\/Taps\/homebrew}
```

Comment:
Don't think `HOMEBREW_LIBRARY` will be set when using `rubocop` outside of `brew style`?